src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pxd

```diff
@@ -37,3 +37,5 @@ cdef class CompletionQueue:
   cdef bint is_polling
   cdef bint is_shutting_down
   cdef bint is_shutdown
+
+  cdef interpret_event(self, grpc.grpc_event event)
```

Comment:
True, but it would still communicate the internalness of this method to readers of the code. They're a valid audience too, right?